internal/render/render-template.subtests.go

```diff
@@ -0,0 +1,29 @@
+// +build go1.7
+
+package render
+
+import (
+	"io"
+
+	"github.com/cweill/gotests/internal/models"
+)
+
+func Header(w io.Writer, h *models.Header) error {
+	if err := tmpls.ExecuteTemplate(w, "header", h); err != nil {
+		return err
+	}
+	_, err := w.Write(h.Code)
+	return err
+}
+
+func TestFunction(w io.Writer, f *models.Function, printInputs bool) error {
+	return tmpls.ExecuteTemplate(w, "function", struct {
+		*models.Function
+		PrintInputs bool
+		Subtests    bool
```

Comment:
New subtest implementation passes a Subtests bool always set to true for Go 1.7.